test/studies/shootout/k-nucleotide/benmcd/knucleotide-user-hash.chpl

```diff
@@ -0,0 +1,149 @@
+use IO, Map, Sort;
+
+config param tableSize = 2**16,
```

Comment:
I agree with you about removing tableSize in what you submit.  There's no need to update the Chapel repo simultaneously, just make sure not to submit code that doesn't work.  Once the submission is accepted, we'll want to reflect it in the test/studies/shootout/submitted/ directory anyway, so that version will reflect what's on the CLBG site.